rclcpp/src/rclcpp/executor.cpp

```diff
@@ -38,14 +41,25 @@
 
 using namespace std::chrono_literals;
 
-using rclcpp::exceptions::throw_from_rcl_error;
 using rclcpp::Executor;
+class rclcpp::ExecutorImplementation {};
```

Comment:
how is this going to be used?

Reply:
We weren't comfortable merging the `rclcpp::WaitSet` implementation with the few outstanding test failures on Windows, this gives a little bit of latitude to backport without breaking ABI into Iron.